driver_test.go

```diff
@@ -3367,9 +3367,12 @@ func TestConnectionAttributes(t *testing.T) {
 
 	attr1 := "attr1"
 	value1 := "value1"
-	attr2 := "foo"
-	value2 := "boo"
-	dsn += fmt.Sprintf("&connectionAttributes=%s:%s,%s:%s", attr1, value1, attr2, value2)
+	attr2 := "fo/o"
+	value2 := "bo/o"
+	dsn += fmt.Sprintf(
+		"&connectionAttributes=%s:%s,%s:%s",
+		attr1, value1, url.QueryEscape(attr2), url.QueryEscape(value2),
```

Comment:
Change to:
```go
dsn += "&connectionAttributes=" + url.QueryEscape(fmt.Sprintf("%s:%s,%s:%s", attr1, value1, attr2, value2))
```

Because the QueryUnescape is applied to the whole value.

Reply:
Nope. Although the implementation is applied to the whole value, but we don't expect users to do that.

Reply:
I'm sorry but in fact we expect users to use the syntax that is supported.

Reply:
Well, *I*'m expecting users to do that.

Reply:
I think @dolmen is right here. Please fix it.